src/components/grid.css

```diff
@@ -1,68 +1,67 @@
 .outerWrapper {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(15em, 1fr));
-    grid-gap: 1rem;
-    grid-template-areas: 
-        "header"
-        "nav"
-        "page-content"
-        "footer";
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(15em, 1fr));
+  grid-gap: 1rem;
+  grid-template-areas: 
+    "header"
+    "nav"
+    "page-content"
+    "footer";
 }
 header {
-    grid-area: header;
+  grid-area: header;
 }
 nav {
-    grid-area: nav;
+  grid-area: nav;
 }
 main {
-    grid-area: page-content;
+  grid-area: page-content;
 }
 footer {
-    grid-area: footer;
+  grid-area: footer;
 }
 /* Media queries for grid layouts */
 @media screen and (min-width: 45em) {
-    .page-people > section {
-        display: grid;
-        grid-template-columns: repeat(2, 1fr);
-    }
-    .page-people > section > h1 {
-        grid-column: 1 / 3;
-    }
+  .page-people > section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 1.5rem;
+  }
+  .page-people > section > h1 {
+    grid-column: 1 / 3;
+  }
 }
 @media screen and (min-width: 61em) {
-    header,
-    nav,
-    footer,
-    .page-home,
-    .page-people,
-    .page-person,
-    .page-news,
-    section.post {
-        display: grid;
-        grid-template-columns: repeat(12, 1fr);
-        gap: 1rem;
-    }
-    .logo,
-    .navbar,
-    .small-print,
-    .leader,
-    section.person,
-    .news,
-    section.post > .post-title,
-    section.post > .post-meta,
-    section.post > .post-content {
-        grid-column: 2 / 10;
-    }
-    .pagination {
-        grid-column: 2 / 12;
-    }
-    .page-people > section {
-        display: grid;
-        grid-column: 2 / 12;
-        grid-template-columns: repeat(3, 1fr);
-    }
-    .page-people > section > h1 {
-        grid-column: 1 / 4;
-    }
+  header,
+  nav,
+  footer,
+  #main-content > div,
```

Comment:
If I am reading this correctly, this just substitutes a more terse selector for the same set of elements. I agree that the current version is verbose—my thinking was that when we felt confident about the set of things we wanted to style this way we could substitute in a (re)created class.

I am fine with this way of doing instead but I worry a little bit that this less specific selection may bite us down the line.

Reply:
Fair point. I would hope that we are consistent enough with the structure of the pages that it wouldn't be a problem as we add more pages, but I am totally fine with substituting at a later date.

Reply:
I hope we're that consistent too :)

Reply:
Another alternative is to use a universal class like `container` that we could apply to any element that needs to have a grid layout. Wrappers and containers are pretty standard when it comes to css grid layouts. Would keep the CSS succinct, but allow us to add/remove elements without needing to adjust the CSS.

Reply:
Yes, a universal class is what I would like to move to